servicebus/data-plane/azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ConfigValidateTestBase.java

```diff
@@ -0,0 +1,21 @@
+package com.microsoft.azure.servicebus;
+
+import org.junit.Assume;
+import org.junit.BeforeClass;
+
+public class ConfigValidateTestBase {
+    static final String RECORD_MODE = "RECORD";
+
+    @BeforeClass
+    public static void skipIfNotConfigured() {
+        Assume.assumeTrue("The test only runs in Live mode.", ConfigValidateTestBase.getTestMode().equalsIgnoreCase(ConfigValidateTestBase.RECORD_MODE));
+    }
+
+    public static String getTestMode(){
+        String testMode =  System.getenv("AZURE_TEST_MODE");
```

Comment:
Instead of defining your own, there is a TestMode enum, you can use in: azure-common-test. The class is: $\common\azure-common-test\src\main\java\com\azure\common\test\TestMode.java.
Then, you don't have to do the string compare above.

Reply:
updated